homeassistant/components/elmax/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "elmax",
+  "name": "elmax-cloud",
```

Comment:
```suggestion
  "name": "Elmax",
```

Reply:
I shall agree.

The reason why I've initially named this extension as "elmax-cloud" is because I was thinking to develop two distinct integrations: one for cloud polling and another one for local polling. 
However, after discussing this with the Elmax engineers, we realized it'll better to have a single integration which is capable of discovering local devices and handle them accordingly.

@frenck : is an integration allowed to declare its class as "CLOUD POLLING" and then discover local deviecs via mdns/zeroconf and handle them differently? In other words, can we have an integration that handles both cloud and lan-local devices at the same time?

Reply:
> The reason why I've initially named this extension as "elmax-cloud" is because I was thinking to develop two distinct integrations: one for cloud polling and another one for local polling.

Those should be merged in a single integration.

Reply:
Makes perfect sense to me. So, once we implement the "local polling", we shall change the IoT class. For now, that's out-of-scope, so the only possibility is to rely on the cloud.

Thanks!